.github/workflows/test-linux-cpu.yml

```diff
@@ -13,58 +13,50 @@ env:
   CHANNEL: "nightly"
 
 jobs:
+  setup:
```

Comment:
Do we still need this `setup` job?

Reply:
Oh I didn't see the repository input for the generic job. Yeah we can use that instead.

Reply:
nit: feedback on the inputs for the linux job - if the primary target for this is CI/lint jobs as opposed to Release/Nightly builds (which are covered by our other Nova tooling), I feel the ref argument should default to main instead of nightly